From: Mike Rapoport <rppt@linux.ibm.com>
Subject: loongarch: drop definition of PGD_ORDER
Date: Tue, 5 Jul 2022 18:47:05 +0300

drop extra BLANK() line in arch/loongarch/kernel/asm-offsets.c

Link: https://lkml.kernel.org/r/20220705154708.181258-13-rppt@kernel.org
Signed-off-by: Mike Rapoport <rppt@linux.ibm.com>
Acked-by: Huacai Chen <chenhuacai@kernel.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 arch/loongarch/kernel/asm-offsets.c |    1 -
 1 file changed, 1 deletion(-)

--- a/arch/loongarch/kernel/asm-offsets.c~loongarch-drop-definition-of-pgd_order-v2
+++ a/arch/loongarch/kernel/asm-offsets.c
@@ -190,7 +190,6 @@ void output_mm_defines(void)
 #endif
 	DEFINE(_PTE_T_LOG2, PTE_T_LOG2);
 	BLANK();
-	BLANK();
 	DEFINE(_PMD_SHIFT, PMD_SHIFT);
 	DEFINE(_PGDIR_SHIFT, PGDIR_SHIFT);
 	BLANK();
_
